gemini/sample-apps/fixmycar/cloud-sql/pgvector/generate-embeddings.ipynb

```diff
@@ -0,0 +1,425 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    PROJECT_ID = 'PROJECT-ID'
</pre><p>Also create a constant for LOCATION/region and use it in the vertexai.init function.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/458/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>